Assignment1/q1.rb

```diff
@@ -0,0 +1,19 @@
+puts "Enter number"
+
+number = gets.to_i
+flag = 0
+if number > 1
+  for i in 2 ... number do
+
+   if number % i == 0
+    flag = 1
```

Comment:
Here flag is not required, you can remove it and can check i == number ? "Prime" : "Not Prime"